<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JirachiAdvancer</class>
 <widget class="QDialog" name="JirachiAdvancer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Jirachi Advancer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelStartingSeed">
     <property name="text">
      <string>Starting Seed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="TextBox" name="textBoxStartingSeed"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="labelTargetSeed">
     <property name="text">
      <string>Target Seed</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="TextBox" name="textBoxTargetSeed"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelMaxAdvances">
     <property name="text">
      <string>Max Advances</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="TextBox" name="textBoxMaxAdvances"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="labelBruteForceRange">
     <property name="text">
      <string>Brute Force Range</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="TextBox" name="textBoxBruteForceRange"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
